common/locales/en/merch.json

```diff
@@ -0,0 +1,10 @@
+{
+  "merch" : "Merchandise",
+
+  "merch-0-summary" : "Teespring is a platform that makes it easy for anyone to create and sell high-quality products people love, with no cost or risk.",
```

Comment:
I did something like this for the FAQ strings and I am now regretting it. It locks you into a particular order, and it makes it hard to retire a piece of merchandise. 

I propose something like: 

`merch-teespring-summary`